src/objc/abstract/MCOAbstractPart.h

```diff
@@ -109,4 +109,8 @@ namespace mailcore {
 
 @end
 
+@interface MCOAbstractPart (MCOUnavailable)
+- (id) init NS_UNAVAILABLE;
```

Comment:
I couldn't find any `MCOAbstractPart` subclasses that needed `-init` called.  If you know of any, please let me know and I'll remove this annotation.